### PR TITLE
Last version for Submit theme

### PR DESCRIPTION
**Pull request by Farnaziifz**
- [ ] Read the [theme publishing doc](https://hexo.io/docs/themes#Publishing) or [plugin publishing doc](https://hexo.io/docs/plugins#Publishing).

### CODE REVIEW

**Comment by YoshinoriN** [comment]
What's purpose of this pull-request ???

File: _config.yml [hunk 1]
```diff
@@ -14,7 +14,7 @@ new_post_name: :year-:month-:day-:title.md # File name of new posts
 post_asset_folder: true
 per_page: 0
 
-theme: navy
+theme: hexo-hive
```
Comment (YoshinoriN) [inline _config.yml]: Why change this line ?

File: .gitmodules [hunk 2]
```diff
@@ -0,0 +1,3 @@
+[submodule "themes/hexo-hive"]
+  path = themes/hexo-hive
```
Comment (YoshinoriN) [inline .gitmodules]: Why add submodule ?

**Comment by Farnaziifz** [comment]
Sry. I made mistake to pull my request.
Plaese close my pull reaquest and i' ll fixed my mistake and pulled again.
Really sry

**Comment by YoshinoriN** [comment]
Okay, No problem.
If you wanna add your theme or plugin, we will wait your contribution,
Please don't hesitate to create pull-request :smile:

Thanks !!

**Comment by YoshinoriN** [comment]
@Farnaziifz 
Btw, if you wanna add theme, please check [publishing doc](https://hexo.io/docs/themes#Publishing).

**Comment by YoshinoriN** [comment]
OK. You need `a15fbc0` commit only ?

commit

**Comment by Farnaziifz** [comment]
Yes please,
Thanks a lot

**Comment by YoshinoriN** [comment]
It seems you are already re-fork [hexo/site](https://github.com/hexojs/site).

site

Are you not familiar with git ?
If yes, I think, once dispose your local machine hexo/site repository. And re-clone current your [forked hexo/site](https://github.com/Farnaziifz/site) repository is better.

**Comment by YoshinoriN** [comment]
If you will not solve problem, please join [gitter room](https://gitter.im/hexojs/hexo).
Thanks.

**Comment by Farnaziifz** [comment]
Sure. Thanks
I send a new pull request. Can you check again please??

**Comment by YoshinoriN** [comment]
Hmm ? Did you really create new pull-request ??
Could you please confirm ?

**Comment by Farnaziifz** [comment]
Pull request ID is #807
Please check,
Thanks  a lot

**Pull request closed without merging**